java/core/src/java/org/apache/orc/impl/MemoryManagerImpl.java

```diff
@@ -81,6 +81,7 @@ public Thread getOwner() {
    */
   public MemoryManagerImpl(Configuration conf) {
     double maxLoad = OrcConf.MEMORY_POOL.getDouble(conf);
+    ROWS_BETWEEN_CHECKS = OrcConf.ROWS_BETWEEN_CHECKS.getLong(conf);
```

Comment:
I don't think we want to support too larger interval for this. Having very high value means prolonging the memory check which is bad (flush often as opposed to don't flush and fail). 
1 to 10000 may be good range. Also please make a note in the description that keeping too low value is for testing only and can cause too early flushes in some cases and generate sub-optimal orc files.